src/api/routers/participantsRouter.ts

```diff
@@ -277,6 +277,15 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.put(
+    '/:participantId/completeRecommendations',
+    async (req: ParticipantRequest, res: Response) => {
+      const { participant } = req;
+      await participant!.$query().update({ completedRecommendations: true });
+      return res.status(200).json(participant?.completedRecommendations);
```

Comment:
I think it's a bit weird to return a boolean, maybe better to return the updated participant or just 200?

Reply:
fair enough, just returning 200 now

Reply:
returning the updated participant now :)